papergrid/tests/extract.rs

```diff
@@ -18,7 +18,7 @@ fn extract_inside_test() {
 fn extract_left_test() {
     let grid = new_grid::<3, 3>();
 
-    let grid = grid.extract(.., ..1);
+    let grid = grid.extract(&(..), &(..1));
```

Comment:
Can you remove these `&` references I am pretty sure it will work out?

Reply:
Now I see the issue.
Look how I fixed it.
Maybe there's some better way but this is OK I think.

https://github.com/zhiburt/tabled/blob/0cb4d9dc0a4ee2c697f57783999fb2fd942abceb/src/extract.rs#L170-L178